### PR TITLE
Refactor to diminish duplicated code

### PR DESCRIPTION
**Pull request by shendiaomo**
Encapsulate text-based and image-based `savefig` into a single function in a standalone module. Other framework-independent code would be moved to `explainer.py` later.

### CODE REVIEW

File: python/sqlflow_submitter/explainer.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 The SQLFlow Authors. All rights reserved.
```
Comment (Yancey0623) [inline python/sqlflow_submitter/explainer.py:1]: Maybe we can rename this file to `plotter.py`

**Review by Yancey0623 — COMMENTED** [review]
Add a `explainer_test.py` to test it?

File: python/sqlflow_submitter/explainer.py (hunk 1 above)
Comment (shendiaomo) [reply python/sqlflow_submitter/explainer.py:1]: I plan to add some common explanation code in this file, `explainer.py` would be more appropriate.

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**